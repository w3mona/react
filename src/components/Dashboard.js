import React from 'react';
import ListExpenses from './List';


const DashBoard = () =>(
    <div>
        <h2>Home</h2>
        <ListExpenses/>
    </div>
);
export default DashBoard;