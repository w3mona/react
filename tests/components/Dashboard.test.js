import React from 'react';
import {shallow} from 'enzyme';
import Dashboard from '../../src/components/Dashboard';

test("not found" , ()=>{
   const wrapper = shallow(<Dashboard/>);
    expect(wrapper).toMatchSnapshot();
});