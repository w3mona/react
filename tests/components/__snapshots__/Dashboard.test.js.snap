// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not found 1`] = `
<div>
  <h2>
    Home
  </h2>
  <Connect(ListExpenses) />
</div>
`;
